tests/helpers/test_groq_stream.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import sys
+import os
+import unittest
+from groq import AsyncGroq
+from dotenv import dotenv_values
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../..")))
+
+from iftk.helpers.groq_stream import GroqProcessor
+
+DOTENV = dotenv_values(".env")
+GROQ_API_KEY = DOTENV["GROQ_API_KEY"]
+messages = [{"role": "user", "content": "How are you doing?"}]
+model_id = "llama-3.1-70b-versatile"
+client = AsyncGroq()
```

Comment:
client creation should be within the test function. Either direct in the test, or, if needing to use it for every test, in the `setUp` method within `TestGroq`.

```
    def setUp(self):
        self.client = AsyncGroq()
```